src/test/java/Homework18Test.java

```diff
@@ -16,9 +16,8 @@ public void playSong(){
         clickSubmit();
         WebElement avatarIcon = driver.findElement(By.cssSelector("[alt='Avatar of student']"));
         assertTrue(avatarIcon.isDisplayed());
-        WebElement playSong = driver.findElement(By.xpath("//span[@class='play']"));
-        playSong.click();
-
+        WebElement player = driver.findElement(By.xpath("//span[@class ='play']"));
+        player.isSelected();
```

Comment:
Player. IsSelected(); is not validating that song is playing. Try to write the logic of how to check if the song is playing